k8s/hpa.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: autoscaling/v2
```

Comment:
Does this autoscaler also scale Grafana or just the serve model API?

Reply:
It only scales the inference server